be/src/vec/exec/scan/vscan_node.cpp

```diff
@@ -966,6 +966,24 @@ Status VScanNode::_normalize_compound_predicate(
                             },
                             active_range);
 
+                    _compound_value_ranges.emplace_back(active_range);
+                }
+            } else if (TExprNodeType::MATCH_PRED == child_expr->node_type()) {
+                SlotDescriptor* slot = nullptr;
+                ColumnValueRangeType* range_on_slot = nullptr;
+                if (_is_predicate_acting_on_slot(child_expr, in_predicate_checker, &slot,
```

Comment:
not match_predicate_checker?

Reply:
match_checker in `_normalize_match_predicate`